### ISSUE TITLE
Inconsistence between paper and training scripts on NMT tasks

### ISSUE THREAD

**Issue by yzlnew** [amirgholami/adahessian#25]
On page 16 of [the newest version](https://arxiv.org/pdf/2006.00719.pdf), it mentions that,
    
>  We set dropout as 0.0 for Transformer base/small model.

However `--dropout 0.3` is used in https://github.com/amirgholami/adahessian/blob/d1a34422f8e07ee18baca9b71827cfcc84978435/transformer/config/adahessian.sh#L22

More importantly, for `lr` of `AdamW`, the paper adopts lr from [this work](https://arxiv.org/pdf/1906.01787.pdf), which utilizes 

> lr =7×10−4/5×10−4 for Transformer-Base/Big respectively

While `lr=0.0015` is used in https://github.com/amirgholami/adahessian/blob/d1a34422f8e07ee18baca9b71827cfcc84978435/transformer/config/adam.sh#L17

Would be grateful if the original training parameters are provided for reproducing results.
